doc/support.rst

```diff
@@ -102,20 +102,23 @@ questions.
 Gitter
 ======
 
-**Note**: The scikit-learn Gitter room is no longer an active community. 
-For live discussions and support, please refer to the other channels 
+**Note**: The scikit-learn Gitter room is no longer an active community.
+For live discussions and support, please refer to the other channels
 mentioned in this document.
 
 .. _documentation_resources:
 
 Documentation Resources
 =======================
 
-This documentation is for |release|. Find documentation for other versions 
+This documentation is for |release|. Find documentation for other versions
 `here <https://scikit-learn.org/dev/versions.html>`__.
 
-Older versions' printable PDF documentation is available `here
-<https://sourceforge.net/projects/scikit-learn/files/documentation/>`_.
-Building the PDF documentation is no longer supported in the website,
-but you can still generate it locally by following the
-:ref:`building documentation instructions <building_documentation>`.
+In case offline access is important, you can download a zipped bundle of the
+documentation.
+
+If you want a PDF, we don't provide one anymore, but you can still generate it
+locally by following the :ref:`building documentation instructions
```

Comment:
I feel a bit the wording here can be improved? How about something in the line of 
> We no longer provide a PDF version of the documentation, but you can still generate it locally by following the :ref:`building documentation instructions<building_documentation>`.

Reply:
Basically I was trying to convey this:
- all the versions of the doc are here
- if you want offline here is zipped HTML
- if you really want a PDF, we don't provide one any more but you can generate it yourself or look get the one for this very old version 

But yeah maybe simplifying the wording is the right call here :sweat_smile:

Reply:
I don't mean removing the other paragraph, just rewording the one regarding the pdf version.